source/index.md

```diff
@@ -2,13 +2,7 @@
 title: API Reference
 
 language_tabs:
```

Comment:
Ok. I thought that in what language the API was made, but is in what language the user can access the API.